expectation.go

```diff
@@ -41,6 +41,12 @@ type Expectation struct {
 	// Times expected to be called
 	minCalls, maxCalls int
 
+	// stdin expectation, as a string or a Matcher
+	stdin interface{}
+
+	// A copy of the stdin data read by the call
+	readStdin []byte
+
```

Comment:
I wasn't sure about naming, but `stdin` takes precedent from `arguments` really meaning “expected arguments”; we're in an Expectation struct, so I guess it's implicit.

And `readStdin` takes precedent from `writeStdout` etc.